about/build-strategies.adoc

```diff
@@ -1,13 +1,12 @@
 :_content-type: ASSEMBLY
-[id="installing-sample-build-strategies"]
-= Installing build strategies
+[id="build-strategies"]
+= Build strategies
 include::_attributes/common-attributes.adoc[]
-:context: installing-sample-build-strategies
+:context: build-strategies
 
 toc::[]
 
-You can install a build strategy or cluster build strategy on the {ocp-product-title} cluster. A `ClusterBuildStrategy` resource is available cluster-wide and a `BuildStrategy` resource is available within a namespace.
-
+You can use a curated set of build strategies or cluster build strategies on the {ocp-product-title} cluster. The {builds-operator} automatically installs these strategies for use. This automated installation of build strategies helps you to quickly get started with {builds-shortname}.
```

Comment:
```suggestion
You can use a curated set of build strategies or cluster build strategies on the {ocp-product-title} cluster. The {builds-operator} automatically installs these strategies for use. This automated installation of cluster build strategies helps you to quickly get started with {builds-shortname}.
```
As we'll have the strategies installed at cluster-scope by operator, I think it'll be better to specify the cluster build strategies here. WDYT?